### PR TITLE
Read cluster list

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Ability to read cluster list from path or from request body.

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

N/A at this moment

### CODE REVIEW

File: server/router_utils.go [hunk 1]
```diff
@@ -147,6 +153,44 @@ func readOrgID(writer http.ResponseWriter, request *http.Request) (types.OrgID,
 	return types.OrgID(orgID), true
 }
 
+// readClusterList retrieves list of clusters from request's path
```
Comment (dpensi) [inline server/router_utils.go:156]: the function name is readClusterListFromPath not readClusterList

**Review by joselsegura — APPROVED** [review]
LGTM

File: server/router_utils.go [hunk 2]
```diff
@@ -147,6 +153,44 @@ func readOrgID(writer http.ResponseWriter, request *http.Request) (types.OrgID,
...
+	}
+
+	// split the list into items
+	clusterList := strings.Split(rawClusterList, ",")
+
+	// everything seems ok -> return list of clusters
+	return clusterList, true
+}
+
+// readClusterList retrieves list of clusters from request's body
```
Comment (dpensi) [inline server/router_utils.go:178]: there are two comments, here and at line that are the same, is this correct?

Comment (tisnik) [reply server/router_utils.go]: good catch, fixed!

**Comment by dpensi** [comment]
LGTM

**Review by dpensi — APPROVED** [review]

**Review by Bee-lee — APPROVED** [review]

**Pull request merged**